### PR TITLE
Update 03 file

### PR DESCRIPTION
**Pull request by beachside-project-ghost**
This pull request includes a change to the `_slides/03-beachside-project-ghost.md` file. The caption text has been updated from "Caption here" to "Hello I'm BEACHSIDE PROJECT GHOST!!".

### CODE REVIEW

**Review by beachside-project — COMMENTED** [review]
LGTM...

**Comment by beachside-project-ghost** [comment]
Are you approved ?

**Review by beachside-project — CHANGES_REQUESTED** [review]
i want request changes

File: _slides/03-beachside-project-ghost.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello I'm BEACHSIDE PROJECT GHOST!!
```
Comment (beachside-project) [inline _slides/03-beachside-project-ghost.md:6]: More funny ?

Comment (beachside-project) [reply _slides/03-beachside-project-ghost.md:6]: ok

**Review by beachside-project — APPROVED** [review]

File: _slides/03-beachside-project-ghost.md (hunk 1 above)
Comment (beachside-project-ghost) [reply _slides/03-beachside-project-ghost.md:6]: ok

**Pull request merged**